hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -1263,8 +1263,8 @@ private OzoneFileStatus getOzoneFileStatusFSO(OmKeyArgs args,
         return new OzoneFileStatus();
       }
 
-      fileStatus = OMFileRequest.getOMKeyInfoIfExists(metadataManager,
-              volumeName, bucketName, keyName, scmBlockSize);
+      fileStatus = OMFileRequest.getOMKeyInfoIfExists(ozoneManager,
+          metadataManager, volumeName, bucketName, keyName, scmBlockSize);
```

Comment:
not a major comment but we can pass the  default replicationConfig object here instead of the ozoneManager instance as it is used only to compute that one member variable just as below in OzoneListStatusHelper constructor.

Reply:
Good point, updated.